src/storage/slab/slab.h

```diff
@@ -129,7 +129,8 @@ struct slab {
     TAILQ_ENTRY(slab) s_tqe;        /* link in slab lruq */
 
     uint8_t           id;           /* slabclass id */
-    uint32_t          padding:24;   /* unused */
+    uint8_t           initialized;  /* flag is slab initialized*/
+    uint16_t          padding;      /* unused */
```

Comment:
`uint16_t unused; /* unused, must be 0 */`

And zero this field during initialization.

Reply:
Done